Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/QuotaCreatedForTierIntegrationEventHandlerTests.cs

```diff
@@ -14,29 +14,24 @@ namespace Backbone.Modules.Quotas.Application.Tests.Tests.Identities;
 
 public class QuotaCreatedForTierIntegrationEventHandlerTests
 {
-    private IMetricStatusesService _metricStatusesService;
-
     [Fact]
     public async void Creates_tier_quota_after_consuming_integration_event()
     {
         // Arrange
         var tierId = new TierId("TIRFxoL0U24aUqZDSAWc");
-        var tier = new Tier(tierId, "some-tier-name");
 
-        var max = 5;
-        var period = QuotaPeriod.Month;
-        var tierQuotaDefinition = new TierQuotaDefinition(MetricKey.NumberOfSentMessages, max, period);
+        var tierQuotaDefinition = new TierQuotaDefinition(MetricKey.NumberOfSentMessages, 5, QuotaPeriod.Month);
         var tierQuotaDefinitionsRepository = new FindTierQuotaDefinitionsStubRepository(tierQuotaDefinition);
 
         var firstIdentity = new Identity("some-identity-address-one", tierId);
         var secondIdentity = new Identity("some-identity-address-two", tierId);
         var identities = new List<Identity> { firstIdentity, secondIdentity };
         var identitiesRepository = A.Fake<IIdentitiesRepository>();
         A.CallTo(() => identitiesRepository.FindWithTier(tierId, CancellationToken.None, true)).Returns(identities);
-        var handler = CreateHandler(identitiesRepository, tierQuotaDefinitionsRepository);
+        var handler = CreateHandler(identitiesRepository, tierQuotaDefinitionsRepository, null);
```

Comment:
Oh come on... Just make the parameter optional instead of only nullable... ;)

Same for the other file